rulesets/test/serialization/sps-request-support-json.test.js

```diff
@@ -0,0 +1,114 @@
+            post:
+                summary: Create User
+                requestBody:
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    id:
+                                        type: string
+                                        description: The unique identifier for the system this user belongs to.
+                                        format: uid
+                                        example: 12345678-1234-1234-1234-123456789012
+                    required: true
+                responses:
+                    "201":
+                        description: User has been successfully created.
+        `;
+    
+        await spectral.validateSuccess(spec, ruleName);
+    });
+
+    test("no request parameters validate", async () => {
+        const spec = `
+        openapi: 3.0.1
+        paths:
+          /v1/users/{id}:
+            post:
+                summary: Create User
+                responses:
+                    "201":
+                        description: User has been successfully created.
+        `;
+    
+        await spectral.validateSuccess(spec, ruleName);
+    });
+
+    test("multiple content types works with application/json", async () => {
+        const spec = `
+        openapi: 3.0.1
+        paths:
+            /v1/users:
+                post:
+                    summary: Create User
+                    requestBody:
+                        content:
+                            multipart/form-data: 
+                                schema:
+                                    type: object
+                                    properties:
+                                        id:
+                                            type: string
+                                            description: The unique identifier for the system this user belongs to.
+                                            format: uid
+                                            example: 12345678-1234-1234-1234-123456789012
+                            application/json:
+                                schema:
+                                    type: object
+                                    properties:
+                                        id:
+                                            type: string
+                                            description: The unique identifier for the system this user belongs to.
+                                            format: uid
+                                            example: 12345678-1234-1234-1234-123456789012
+                        required: true
+                    responses:
+                        "201":
+                            description: User has been successfully created.
+        `;
+        await spectral.validateSuccess(spec, ruleName);
+    });
```

Comment:
Perfect test for validating the bug fix works as expected

Reply:
ya for testing :)